NEWS.md

```diff
@@ -602,10 +602,10 @@ specific for admiral. Derivations like this can be implemented calling
 - Additional explanation added to `derive_param_*` and `derive_derived_param` functions regarding which variables are populated in the additional rows (#939)
 
 
-- Updated [`derive_var_worst_flag()`](https://pharmaverse.github.io/admiral/reference/derive_var_worst_flag.html) and [`derive_var_extreme_flag()`](https://pharmaverse.github.io/admiral/reference/derive_var_extreme_flag.html) vignettes to clarify their purpose (#691)
+- Updated [`derive_var_worst_flag()`](https://pharmaverse.github.io/admiral/cran-release/reference/derive_var_worst_flag.html) and [`derive_var_extreme_flag()`](https://pharmaverse.github.io/admiral/cran-release/reference/derive_var_extreme_flag.html) vignettes to clarify their purpose (#691)
```

Comment:
I think we do not need to link the functions explicitly. Using just `` `derive_var_worst_flag()` `` and `` `derive_var_extreme_flag()` ``. should work. And it has the advantage that we do not get a broken link if `derive_var_worst_flag()` is removed in 0.12.0.